---
hal_ugv/State state
